### DISCUSSION TITLE
model warzone 2

### DISCUSSION THREAD

**Discussion by achille12344** [Babyhamsta/Aimmy#77] [General]
Is it possible to have a model for warzone 2 thanks in advance

**Comment by achille12344** [comment]
please can someone post the warzone model thank you very much

**Comment by Alex-test-mod** [comment]
hi you need model warzone 2 ??

**Reply by achille12344** [reply]
yes bro

**Reply by achille12344** [reply]
hi brother or tried modell for warzone but it doesn't work

**Comment by Alex-test-mod** [comment]
model warzone 2 train

![labels](https://github.com/Babyhamsta/Aimmy/assets/34565256/7a3455f6-d911-4134-9440-45710c422d00)

![labels_correlogram](https://github.com/Babyhamsta/Aimmy/assets/34565256/8e3151ba-0d18-486e-b62c-10bc60f32a0e)

**Reply by achille12344** [reply]
brother tried it and this model doesn't work

**Reply by Alex-test-mod** [reply]
need mor photos

**Reply by Alex-test-mod** [reply]
I need at least 1000 photos for it to work properly

**Reply by Alex-test-mod** [reply]
1920 1080

**Reply by achille12344** [reply]
yes bro

tried it and this model doesn't work
